### PR TITLE
Try GitHub Actions CI

### PR DESCRIPTION
**Pull request by cclauss**
From https://github.com/actions/starter-workflows/blob/master/ci/python-package.yml

### CODE REVIEW

File: .github/workflows/python-package.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [2.7, 3.5, 3.6, 3.7]
```
Comment (hugovk) [inline .github/workflows/python-package.yml]: We only need 3.6+

**Comment by cclauss** [comment]
There is this really weird thing where actions do not work on forks but only on the source repo.  I think if you either land these PRs or you recreate them as repo owner then they work but not on forks because that "pull" message is not received.

It would be great if you could help me grok that.

**Comment by cclauss** [comment]
Nope.  Inexplicable.

**Comment by cclauss** [comment]
Now they ran (see above)

**Comment by cclauss** [comment]
Closing in favor of #70

**Comment by hugovk** [comment]
Yep, your theory seems correct. Just needs some tweaks to install requirements to make it pass. I'll check it later, and probably make it use tox like AP. Thanks!

**Comment by cclauss** [comment]
If you ever have spare cycles to help on a port to Python 3, checkout nodejs/node.

**Pull request closed without merging**